.gitignore

```diff
@@ -8,6 +8,7 @@ result
 *.pending-snap
 *.snap*
 !cli/tests/cli-reference@.md.snap
+.cargo/insta.yaml
```

Comment:
Can you add some comment saying this is per-user configuration?

Reply:
Done.